libs/platform/src/lib/components/form/datetime-picker/datetime-picker/datetime-picker.component.ts

```diff
@@ -0,0 +1,348 @@
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    EventEmitter,
+    Host,
+    Input,
+    OnDestroy,
+    OnInit,
+    Optional,
+    Output,
+    Self,
+    SkipSelf,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import { NgControl, NgForm } from '@angular/forms';
+import {
+    CalendarYearGrid,
+    DatetimePickerComponent,
+    DaysOfWeek,
+    FdCalendarView,
+    FdDate,
+    FdDatetime,
+    SpecialDayRule
+} from '@fundamental-ngx/core';
+import { Placement } from 'popper.js';
+import { BaseInput } from '../../base.input';
+import { FormField } from '../../form-field';
+import { FormFieldControl, Status } from '../../public_api';
+
+@Component({
+    selector: 'fdp-datetime-picker',
+    templateUrl: './datetime-picker.component.html',
+    styleUrls: ['./datetime-picker.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None,
```

Comment:
`datetime-picker.component.scss` have no styles, so it could be removed 

Also, `encapsulation: ViewEncapsulation.None` could be removed, since we have no styles